Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6543,7 +6543,7 @@ WebGLEnabled:
 
 WebGLUsingMetal:
   type: bool
-  status: testable
+  status: internal
```

Comment:
Note that this change means that this toggle will not be flipped on automatically for test runs; it seems like this isn't an issue, since the feature will be enabled by default on platforms that support WebGL via Metal.